src/components/ex-title/ex-title.scss

```diff
@@ -0,0 +1,33 @@
+.title {
+  .cmp-title {
+    scroll-margin-top: 100px;
+
+    &__text,
+    &__link {
+      @apply text-primary text-l font-headline mb-0 font-light;
+
+      hyphens: auto;
+    }
+
+    &__link {
+      &__text {
+        &:hover {
+          @apply text-interactive-state-hover-link underline;
+        }
+      }
+    }
+
+    &__read-more {
```

Comment:
Is there a read more button in the HTML template of the SichereArbeit hero story? If not, this class can be removed.

Reply:
Removed read more class